less/common.less

```diff
@@ -163,8 +163,8 @@ html.no-js .no-js-warning {
   padding-left: 2rem;
   li {
     &:before {
-      content: "\f363";
-      font-family: "Ionicons";
+      content: "\f0da";
+      font-family: "FontAwesome";
       margin-right: .5rem;
     }
```

Comment:
Carets on the Event Resources page

![screen shot 2015-06-16 at 3 00 04 pm](https://cloud.githubusercontent.com/assets/2896608/8195573/69ce879e-1438-11e5-9a21-0a1ef3be1352.png)